palantir-java-format/build.gradle

```diff
@@ -30,6 +41,36 @@ dependencies {
     annotationProcessor 'org.derive4j:derive4j'
 }
 
+graalvmNative {
+    binaries {
+        main {
+            javaLauncher = javaToolchains.launcherFor {
+                languageVersion = JavaLanguageVersion.of(23)
+                vendor = JvmVendorSpec.GRAAL_VM
+            }
+            imageName = 'palantir-java-format'
+            mainClass = mainClassName
+            fallback = false
+            sharedLibrary = false
+            configurationFileDirectories.from(file('src/main/resources/META-INF/native-image'))
+            buildArgs.addAll("-H:ConfigurationFileDirectories=${projectDir}/src/main/resources/META-INF/native-image",
```

Comment:
same args as in google-java-format: https://github.com/google/google-java-format/blob/eb0b7682f15ec6a2bd03d12b53bdf951dd924266/core/pom.xml#L282-L295